src/python_testing/TC_BRBINFO_4_1.py

```diff
@@ -57,7 +57,8 @@ def desc_TC_BRBINFO_4_1(self) -> str:
 
     def steps_TC_BRBINFO_4_1(self) -> list[TestStep]:
         steps = [
-            TestStep("0", "Preconditions"),
+            TestStep("0",  "DUT commissioned", is_commissioning=True),
+            TestStep("0a", "Preconditions"),
             TestStep("1a", "TH reads from the ICD the A_IDLE_MODE_DURATION, A_ACTIVE_MODE_DURATION, and ACTIVE_MODE_THRESHOLD attributes"),
             TestStep("1b", "Simple KeepActive command w/ subscription. ActiveChanged event received by TH contains PromisedActiveDuration"),
```

Comment:
While you are touching this file can you add in `_ask_for_vendor_commissioniong_ux_operation` that fabric-admin should include `--icd-registration true`